<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditCalculator</class>
 <widget class="QWidget" name="CreditCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
background: none;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widgetMain" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>160</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
color: rgb(255, 210, 255);
border: 1px solid;
font: 13pt &quot;Anta&quot;;
border-radius: 6px;
border-color: rgb(255, 210, 255);
padding: 0 5px;
background-color: rgb(11, 1, 24);
selection-background-color: rgb(248, 160, 165);
selection-color: black;
background-color: rgb(33, 32, 70);

color: rgb(11, 1, 24);
background-color: rgb(255, 210, 255);
border-color: rgb(74, 71, 158);
}


QComboBox::drop-down:button {
color: rgb(255, 210, 255);
border-top-right-radius: 4px;
border-bottom-right-radius: 4px;
border-style: solid;
border-width: 0px;
border-color: rgb(255, 210, 255);
background-color: rgb(74, 71, 158);
padding: 0px;
width: 20px;
}

QComboBox::down-arrow {
image: url(:/icons/keyboard_arrow_down_yellow.svg);
}

QComboBox::drop-down:hover {
background-color: rgb(169, 167, 215);
border-color: rgb(169, 167, 215);
}

QComboBox::drop-down:pressed {
background-color: rgb(25, 24, 53);
border-color: rgb(25, 24, 53);
}

QComboBox QAbstractItemView {
color: rgb(255, 210, 255);
border: 0px solid;
border-radius: 4px;
border-color: rgba(130, 127, 255, 133);
background-color: rgb(2, 1, 18);
}

QComboBox:item {
color: rgb(255, 210, 255);
}

QComboBox:item:selected {
background-color: rgb(74, 71, 158);
color: rgb(255, 210, 255);
/*background-color: rgb(255, 105, 103);*/
}


QPushButton {
font: 13pt &quot;Anta&quot;;
color: rgb(255, 210, 255);
border-radius: 6px;
border-style: solid;
border-width: 2px;
border-color: rgb(8, 1, 18);
background-color: rgb(74, 71, 158);
}

QPushButton:hover {
background-color: rgb(169, 167, 215);
}

QPushButton:pressed {
background-color: rgb(25, 24, 53);
}

QLineEdit {
color: rgb(255, 210, 255);
border: 1px solid;
font: 13pt &quot;Anta&quot;;
border-radius: 6px;
border-color: rgb(255, 210, 255);
padding: 0 8px;
background-color: rgb(33, 32, 70);/*rgb(11, 1, 24);*/
selection-background-color: rgb(248, 160, 165);
selection-color: black;

color: rgb(11, 1, 24);
background-color: rgb(255, 210, 255);
border-color: rgb(74, 71, 158);
}

QDoubleSpinBox {
color: rgb(255, 210, 255);
border: 1px solid;
font: 13pt &quot;Anta&quot;;
padding: 0 5px;
border-radius: 6px;
border-color: rgb(255, 210, 255);
background-color: rgb(11, 1, 24);
selection-background-color: rgb(248, 160, 165);
selection-color: black;
background-color: rgb(33, 32, 70);

color: rgb(11, 1, 24);
background-color: rgb(255, 210, 255);
border-color: rgb(74, 71, 158);
}


QDoubleSpinBox::up-button {
subcontrol-origin: border;
subcontrol-position: top right; 
border-top-right-radius: 3px;
border-style: solid;
border-width: 0px;
border-color: rgb(74, 71, 158);
background-color:  rgb(74, 71, 158);
image: url(:/icons/add_yellow.svg);
padding: 0px;
margin: 0px;
width: 16px;
height: 15.5px;
 }


QDoubleSpinBox::down-button {
subcontrol-origin: border;
subcontrol-position: bottom right;
border-bottom-right-radius: 3px;
border-style: solid;
border-width: 0px;
border-color: rgb(74, 71, 158);
background-color:  rgb(74, 71, 158);
image: url(:/icons/remove_yellow.svg);
padding: 0px;
margin: 0px;
width: 16px;
height: 15.5px;
 }

QDoubleSpinBox::down-button:hover {
background-color: rgb(169, 167, 215);
}

QDoubleSpinBox::down-button:pressed {
background-color: rgb(25, 24, 53);
}

QDoubleSpinBox::up-button:hover {
background-color: rgb(169, 167, 215);
}

QDoubleSpinBox::up-button:pressed {
background-color: rgb(25, 24, 53);
}

QLabel {
font: 16pt &quot;Anta&quot;;
color: rgb(255, 210, 255);
}

QRadioButton {
font: 13pt &quot;Anta&quot;;
color: rgb(255, 210, 255);
}

QRadioButton::indicator {
width: 10px;
height: 10px;
border-radius: 7px;
}

QRadioButton::indicator:checked {
    background-color:  rgb(243, 89, 97);
    border: 2px solid rgb(255, 210, 255);
}

QRadioButton::indicator:unchecked {
    background-color: rgb(8, 1, 18);
    border: 2px solid rgb(255, 210, 255);
}

QSpinBox {
color: rgb(255, 210, 255);
border: 1px solid;
font: 13pt &quot;Anta&quot;;
padding: 0 5px;
border-radius: 6px;
border-color: rgb(255, 210, 255);
background-color: rgb(11, 1, 24);
selection-background-color: rgb(248, 160, 165);
selection-color: black;
background-color: rgb(33, 32, 70);

color: rgb(11, 1, 24);
background-color: rgb(255, 210, 255);
border-color: rgb(74, 71, 158);
}


QSpinBox::up-button {
subcontrol-origin: border;
subcontrol-position: top right; 
border-top-right-radius: 3px;
border-style: solid;
border-width: 0px;
border-color: rgb(74, 71, 158);
background-color:  rgb(74, 71, 158);
image: url(:/icons/add_yellow.svg);
padding: 0px;
margin: 0px;
width: 16px;
height: 15.5px;
 }


QSpinBox::down-button {
subcontrol-origin: border;
subcontrol-position: bottom right;
border-bottom-right-radius: 3px;
border-style: solid;
border-width: 0px;
border-color: rgb(74, 71, 158);
background-color:  rgb(74, 71, 158);
image: url(:/icons/remove_yellow.svg);
padding: 0px;
margin: 0px;
width: 16px;
height: 15.5px;
 }

QSpinBox::down-button:hover {
background-color: rgb(169, 167, 215);
}

QSpinBox::down-button:pressed {
background-color: rgb(25, 24, 53);
}

QSpinBox::up-button:hover {
background-color: rgb(169, 167, 215);
}

QSpinBox::up-button:pressed {
background-color: rgb(25, 24, 53);
}

QLabel {
color: rgb(255, 210, 255);
font: 14pt &quot;Kode Mono&quot;;
}
</string>
     </property>
     <widget class="QSpinBox" name="spinBoxTime">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>40</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::PlusMinus</enum>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>50</number>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxTimeType">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>40</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="currentText">
       <string>years</string>
      </property>
      <item>
       <property name="text">
        <string>years</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>months</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButtonFillCredit">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>90</y>
        <width>61</width>
        <height>55</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/*QPushButton {
background-color: rgb(0, 179, 116);
}

QPushButton:hover {
background-color: rgb(0, 230, 149);
}

QPushButton:pressed {
background-color: rgb(0, 51, 33);
}*/</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../misc/resource.qrc">
        <normaloff>:/icons/output_yellow.svg</normaloff>:/icons/output_yellow.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxCreditType">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="currentText">
       <string>Annuity</string>
      </property>
      <item>
       <property name="text">
        <string>Annuity</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Differentiated</string>
       </property>
      </item>
     </widget>
     <widget class="QSpinBox" name="spinBoxCredit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::PlusMinus</enum>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1000000000</number>
      </property>
      <property name="stepType">
       <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
      </property>
      <property name="value">
       <number>100000</number>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxRate">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::PlusMinus</enum>
      </property>
      <property name="minimum">
       <double>0.010000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
      <property name="stepType">
       <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
      </property>
      <property name="value">
       <double>10.000000000000000</double>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPayment">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditInterest">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>70</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditTotal">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>120</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelPayment">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>0</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Monthly payment</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelInterest">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>50</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Interest total</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelTotal">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>100</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Debt + interest total</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelCredit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Credit</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelRate">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Rate</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelTerm">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>58</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Term</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelRatePercent">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18px &quot;Kode Mono&quot;;</string>
      </property>
      <property name="text">
       <string>%</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tableWidgetCredit">
     <property name="styleSheet">
      <string notr="true">QTableWidget {
    /*background-color: rgb(2, 1, 18);*/
background-color: rgb(74, 71, 158);
border: 0px inset;
border-color: rgb(2, 1, 18);
    border-radius: 5px;
    font: 12pt &quot;Anta&quot;;
}

QHeaderView {
    background-color: rgb(74, 71, 158);
    padding: 0px;
    border-top-right-radius: 0px;
    border-top-left-radius: 5px;
    border-bottom-right-radius: 0px;
    border-bottom-left-radius: 5px;
    border: 0px solid rgb(74, 71, 158);
    font: 13pt &quot;Anta&quot;;
}

QHeaderView::section {
    background-color: rgb(255, 210, 255);
    border: 1px solid rgb(74, 71, 158);
    color: rgb(11, 1, 24);
}

QHeaderView::section:first {
}

QHeaderView::section:last {

}

QHeaderView::section:pressed {
	background-color: rgb(246, 136, 142);
}

/*QHeaderView::section:checked {
	background-color: rgb(246, 136, 142);
}*/

QTableWidget QTableCornerButton::section {
   background-color: rgb(255, 210, 255);
    border: 1px solid rgb(74, 71, 158);
    color: rgb(74, 71, 158);
    border-top-left-radius: 5px;
}

QTableView::item {
background-color: rgb(74, 71, 158);
    border: 0px solid rgb(74, 71, 158);
    color: rgb(255, 210, 255);
/*
selection-color: rgb(2, 1, 18);
selection-background-color: rgb(243, 89, 97);
*/
}


QTableView::item:selected {
background-color: rgb(246, 136, 142);
    border: 0px solid rgb(74, 71, 158);
color: rgb(2, 1, 18);
}

QScrollBar:vertical {
background-color: rgb(2, 1, 18);
border: 1px solid rgb(74, 71, 158);
color:  rgb(255, 210, 255);
border-width: 0px;border-style: solid;
margin: 25px 0 25px 0;
width: 22px;
}

QScrollBar::handle:vertical {
background-color: rgb(74, 71, 158);
border: 1px solid rgb(74, 71, 158);
image: url(:/icons/menu_yellow.svg);
/*
border-top: 1px solid rgb(2, 1, 18);
border-bottom: 1px solid rgb(2, 1, 18);
border: 1px solid rgb(2, 1, 18);
*/
}

QScrollBar::handle:vertical:hover {
background-color: rgb(169, 167, 215);
border-color: rgb(169, 167, 215);
}

QScrollBar::handle:vertical:pressed {
background-color: rgb(25, 24, 53);
border-color: rgb(25, 24, 53);
}

QScrollBar::add-line:vertical {
border: 1px solid rgb(74, 71, 158);
background-color: rgb(74, 71, 158);
border-bottom-right-radius: 7px;
subcontrol-position: bottom;
subcontrol-origin: margin;
image: url(:/icons/keyboard_arrow_down_yellow.svg);
}

QScrollBar::sub-line:vertical {
border: 1px solid rgb(74, 71, 158);
background-color: rgb(74, 71, 158);
border-top-right-radius: 7px;
subcontrol-position: top;
subcontrol-origin: margin;
image: url(:/icons/keyboard_arrow_up_yellow.svg);
}

QScrollBar::add-line:vertical:hover {
border-color: rgb(169, 167, 215);
background-color: rgb(169, 167, 215);
}

QScrollBar::sub-line:vertical:hover {
border-color: rgb(169, 167, 215);
background-color: rgb(169, 167, 215);
}

QScrollBar::add-line:vertical:pressed {
border-color: rgb(25, 24, 53);
background-color: rgb(25, 24, 53);
}

QScrollBar::sub-line:vertical:pressed {
border-color: rgb(25, 24, 53);
background-color: rgb(25, 24, 53);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideLeft</enum>
     </property>
     <property name="gridStyle">
      <enum>Qt::DotLine</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Month</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Payment</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Main part</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Interest part</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Remainder</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../misc/resource.qrc"/>
 </resources>
 <connections/>
</ui>
